pkg/log/log.go

```diff
@@ -47,22 +47,24 @@ func WithContext(ctx context.Context) {
 }
 
 func (l *logger) Print(msg string) {
+	var message interface{}
```

Comment:
Is this needed?

Reply:
it was failing before with: `pkg/log/log.go:52:14: cannot use msg (type string) as type []interface {} in argument to logrus.Info`

Reply:
OK.

Reply:
Oh I think I know where I went wrong! pushing an updated PR.

Reply:
Sorry for the confusion @julio-lopez, I was calling `logrus.Info(msg...)` instead if `logrus.Info(msg)`. Fixing it now